### PR TITLE
Merge

### PR DESCRIPTION
**Pull request by ocefpaf**
Same as #222 for the `gdal 2.3.1`.

Should fix https://github.com/conda-forge/gdal-feedstock/issues/223

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,67 +1,203 @@
...
   skip: True  # [win and vc<14]
 
 requirements:
   build:
+    - cmake  # [win]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
+    # Python here is only needed so conda-smithy can get the matrix right.
+    # We should probably re-visit this at some point and make it more efficient.
```
[the remark this answers is not in the document]
Comment (isuruf) [reply recipe/meta.yaml]: There shouldn't be any matrix CI files. Libgdal is built once and the python bindings are built for each version. When you add this libgdal is built multiple times wasting CI resources

[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipe/meta.yaml]: > There shouldn't be any matrix CI files. Libgdal is built once and the python bindings are built for each version.

How will the python bindings get built for multiple Pythons in that case?

> When you add this libgdal is built multiple times wasting CI resources

Yep. I know, but we either split into multiple packages or do this. Merging the recipes was the "quick way" to get this recipe looking like the AR.

[the remark this answers is not in the document]
Comment (isuruf) [reply recipe/meta.yaml]: > How will the python bindings get built for multiple Pythons in that case?

conda-build will handle this. You just have to remove python from host here and rerender.

[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipe/meta.yaml]: It did not work last time :-/ but we can always try again...

**Pull request merged**